README.md

```diff
@@ -27,10 +27,10 @@ cd fluxion
 **Fluxion is also available in arch** 
 ```
 cd bin/arch
-makepkg
+make pkg
 ```
 
-or using the blackarch repo
+or using the black arch repo
```

Comment:
The guys over at https://www.blackarch.org/ spell their distribution `BlackArch` without an whitespace.